public class Throw_finaly {

	public static void main(String[] args) {
		int x=12;
		int res,y=0;
		try
		{
			res=x/0;
			System.out.print(res);
			
			//ArithmaticException
		}catch(Exception e)
		{
			
		}

	}

}
